edgraph/server.go

```diff
@@ -1394,7 +1394,12 @@ func processQuery(ctx context.Context, qc *queryContext) (*api.Response, error)
 	// Core processing happens here.
 	er, err := qr.Process(ctx)
 
+	if bool(glog.V(3)) || worker.LogDQLRequestEnabled() {
+		glog.Infof("Finished a query that started at: %+v",
+			qr.Latency.Start.Format(time.RFC3339))
+	}
 	if err != nil {
+		glog.Infof("Error processing query: %+v\n", err.Error())
```

Comment:
Could `err` here contain query processing errors? If so, maybe it should be restricted to the same v3 logging level?  People testing out with different syntax queries could make for some spammy logs.

Reply:
Good idea, gated the logging behind log level.